src/raml2htmlTemplates/resource.nunjucks

```diff
@@ -38,7 +38,7 @@
               <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
               <h4 class="modal-title" id="myModalLabel">
                 <span class="badge badge_{{ method.method }}">{{ method.method }}{% if method.securedBy.length %} <span class="glyphicon glyphicon-lock" title="Authentication required"></span>{% endif %}</span>
-                <span class="parent">{{ resource.parentUrl }}</span>{{ resource.relativeUri }}
+                <span class="break-all"><span class="parent">{{ resource.parentUrl }}</span>{{ resource.relativeUri }}</span>
```

Comment:
As I've mentioned in DevPortal pull request, this class should be called `u-break-all`